services/web/server/tests/integration/01/conftest.py

```diff
@@ -45,7 +45,7 @@ async def user_project(
     fake_project["prjOwner"] = logged_user["name"]
     fake_project["uuid"] = f"{project_id}"
 
-    async with NewProject(
+    async with new_project(
```

Comment:
this change appears not to follow python practices. this should be an object, hence Camelcased